vite.config.ts

```diff
@@ -50,6 +51,7 @@ export default defineConfig(({ mode }) => ({
     ],
   },
   plugins: [
+    nodePolyfills(),
```

Comment:
There was a polyfill error with the `node:crypto` module from the graz library during the build process, so a polyfill has been added to Vite.